director/stencila_open/templates/open/main.html

```diff
@@ -1,12 +1,22 @@
 {% extends 'base.html' %}
+{% load static %}
+
 {% block title %}Open{% endblock %}
 {% block header %}{% include "open/open_header.html" with brand_tagline='/open' navbar_id='navbar_open' display_share=user_owns_conversion %}{% endblock %}
 {% block content %}
     <section class="section">
         <div class="columns">
             <div class="column is-8 is-offset-2">
-                <h1 class="title has-text-centered">Stencila /open</h1>
-                <h2 class="subtitle is-5 has-text-centered">Generate reproducible publication previews, and collaborate with colleagues using other formats.</h2>
+                <h1 class="title is-size-4-mobile has-text-centered">
+                  Reproducible research collaboration &amp; publication, minus&nbsp;all&nbsp;the&nbsp;headache.
```

Comment:
"Reproducible research collaboration & publication, minus the headaches" (remove "all the").